src/main/java/levi9/NotificationService/controller/NotificationController.java

```diff
@@ -0,0 +1,59 @@
+package levi9.NotificationService.controller;
+
+import levi9.NotificationService.service.NotificationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping(value = "/notification",produces = MediaType.APPLICATION_JSON_VALUE)
+public class NotificationController {
+
+
+    @Autowired
+    private final NotificationService notificationService;
+
+    @Autowired
+    public NotificationController(NotificationService notificationService) {
+
+        this.notificationService = notificationService;
+    }
+
+    // ovde cu prosledjivati mejl adresu na koju treba da posaljem verifikaciju
+    @GetMapping("/snsverificationRequest/{email}")
```

Comment:
makes more sense to be /subscription/{email} or /verification/{email}

Reply:
solved